pkg/render/intrusion_detection.go

```diff
@@ -357,6 +365,17 @@ func (c *intrusionDetectionComponent) deploymentPodTemplate() *corev1.PodTemplat
 		}
 	}
 
+	container := ElasticsearchContainerDecorateIndexCreator(
+		ElasticsearchContainerDecorate(c.intrusionDetectionControllerContainer(), c.esClusterConfig.ClusterName(), ElasticsearchIntrusionDetectionUserSecret, c.clusterDomain),
+		c.esClusterConfig.Replicas(), c.esClusterConfig.Shards())
+
+	if c.elasticLicenseType == ElasticLicenseTypeBasic {
+		envVars := []corev1.EnvVar{
+			{Name: "DISABLE_ANOMALY", Value: "yes"},
```

Comment:
Since I don't know what this does can you get someone to review that understands the ML job?

Reply:
I'm comfortable with this change since watchers aren't supported, but do we also need to set DISABLE_ALERTS to true as well?